reactor-core/src/main/java/reactor/util/retry/Retry.java

```diff
@@ -60,6 +68,10 @@
 	 */
 	public abstract Publisher<?> generateCompanion(Flux<RetrySignal> retrySignals);
 
+
+	public ContextView retryContext() {return retryContext;}
```

Comment:
```suggestion
	public ContextView retryContext() {
		return retryContext;
	}
```

Reply:
Could you also please clarify `retryContextView()` vs `retryContext()`?

Reply:
I added a comment on `Retry.retryContext()`.
We could also decide to rename `RetrySignal.retryContextView() -> RetrySignal.retryContext()`, let me know what you think.